### ISSUE TITLE
Changelog fragments

### ISSUE THREAD

**Issue by gardar** [ansible-community/antsichaut#6]
I feel that most of the projects that use `antsibull-changelog` are using it with changelog fragments. It would be useful if antsichaut would support writing changelog fragments and I think it could help with adoption.

**Comment by rndmh3ro** [comment]
Hm, that would mean that there are people or projects, that use manually written changelog fragments and antsichaut-automation? Or are the changelog fragments created automatically somewhere?

**Comment by gardar** [comment]
I looked at quite a few CI workflows when I came across antsichaut. A common CI job for antsibull-changelog seems to be where it's confirmed that the PR contains a fragment file and then antsibull-changelog is used to generate the changelog from it. The fragments themselves always seem to be done manually and antsichaut is the only tool I've seen that's generating the changelogs automatically.

**Comment by rndmh3ro** [comment]
Yes, that's the whole reason antsichaut exists. i didn't want to manually create fragments for every PR (or tell the contributors to do so).

So I'm still not sure how the changelog-fragments should be supported b< antsichaut.

**Comment by gardar** [comment]
Well basically it would just write to a new fragment file for each opened PR instead of appending to the changelogs.yaml file.

**Comment by rndmh3ro** [comment]
I'm still not sure what the benefit of this would be, however I'd  be fine with someone implementing it.

**Comment by gardar** [comment]
It's mostly just different styles, if you've already got a project with fragments you might want to keep using fragments but fill them out automatically with antsichaut.
And fragments is actually what the ansible core project uses: https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments
